test/flows/student_finance_calculator_flow_test.rb

```diff
@@ -0,0 +1,362 @@
+require "test_helper"
+require "support/flow_test_helper"
+
+class StudentFinanceCalculatorTest < ActiveSupport::TestCase
+  include FlowTestHelper
+
+  setup { testing_flow StudentFinanceCalculatorFlow }
+
+  should "render a start page" do
+    assert_rendered_start_page
+  end
+
+  context "question: when_does_your_course_start?" do
+    setup { testing_node :when_does_your_course_start? }
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of what_type_of_student_are_you? for any response" do
+        assert_next_node :what_type_of_student_are_you?, for_response: "2020-2021"
+      end
+    end
+  end
+
+  context "question: what_type_of_student_are_you?," do
+    setup do
+      testing_node :what_type_of_student_are_you?
+      add_responses when_does_your_course_start?: "2020-2021"
+    end
+
+    should "render the question" do
+      assert_rendered_question
+    end
+
+    context "next_node" do
+      should "have a next node of how_much_are_your_tuition_fees_per_year? for any response" do
+        assert_next_node :how_much_are_your_tuition_fees_per_year?, for_response: "uk-full-time"
+      end
+    end
+  end
+
+  context "UK full time students" do
+    context "question: how_much_are_your_tuition_fees_per_year?," do
+      setup do
+        testing_node :how_much_are_your_tuition_fees_per_year?
+        add_responses when_does_your_course_start?: "2020-2021",
+                      what_type_of_student_are_you?: "uk-full-time"
+      end
+
+      should "render the question" do
+        assert_rendered_question
+      end
+
+      context "validation" do
+        should "be invalid if a fee over 9250 is entered for full time students" do
+          assert_invalid_response "9251"
```

Comment:
```suggestion
          max_for_uk_full_time = SmartAnswer::Calculators::StudentFinanceCalculator::TUITION_FEE_MAXIMUM["full-time"]
          assert_invalid_response (max_for_uk_full_time + 1).to_s
```

We can utilise a (admittedly rather long) constant to make this less of a magic number